api/v2.0/swagger.yaml

```diff
@@ -1437,6 +1437,153 @@ paths:
           $ref: '#/responses/404'
         '500':
           $ref: '#/responses/500'
+  /system/gc:
+    get:
+      summary: Get gc results.
+      description: This endpoint let user get gc execution history.
+      tags:
+        - gc
+      operationId: getGCHistory
+      parameters:
+        - $ref: '#/parameters/query'
```

Comment:
Can this API support the `query` directly? Seems the returning `GCHistory` model isn't the underlying database model